docker/docker-bigtable/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '2'
+
+services:
+  metrictank:
+    hostname: metrictank
+    image: grafana/metrictank
+    ports:
+     - "6060:6060"
+     - "2003:2003"
+    volumes:
+      - ../../build/metrictank:/usr/bin/metrictank
+      - ../../scripts/config/metrictank-docker.ini:/etc/metrictank/metrictank.ini
+      - ../../scripts/config/storage-schemas.conf:/etc/metrictank/storage-schemas.conf
+      - ../../scripts/config/storage-aggregation.conf:/etc/metrictank/storage-aggregation.conf
+      - ../../scripts/config/schema-store-cassandra.toml:/etc/metrictank/schema-store-cassandra.toml
+      - ../../scripts/config/schema-idx-cassandra.toml:/etc/metrictank/schema-idx-cassandra.toml
+
+    environment:
+      WAIT_HOSTS: bigtable:8086
+      WAIT_TIMEOUT: 60
+      MT_HTTP_MULTI_TENANT: "false"
+      MT_MEMORY_IDX_TAG_SUPPORT: "true"
+      MT_CASSANDRA_ENABLED: "false"
+      MT_CASSANDRA_IDX_ENABLED: "false"
+      MT_BIGTABLE_STORE_ENABLED: "true"
+      MT_BIGTABLE_IDX_ENABLED: "true"
+      BIGTABLE_EMULATOR_HOST: bigtable:8086
```

Comment:
how do you feel about this variable that bypasses MT's config system? wouldn't it be cleaner to make this an MT config param? i notice that for "real" (non-emulated) bigtable, there is no addr setting though.

Reply:
This is not processed by MT.  This is directly handled by the bigtable client library.  In the same way GOGC is handled directly by the runtime.

Given that this is purely for development, i see no reason to add support for it in the config